manifests/g/GIMP/GIMP/2.10.38.1/GIMP.GIMP.installer.yaml

```diff
@@ -0,0 +1,171 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GIMP.GIMP
+PackageVersion: 2.10.38.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+UpgradeBehavior: install
+FileExtensions:
+- 3fr
+- alpha
+- als
+- apm
+- ari
+- arw
+- avif
+- bay
+- bitmap
+- bmp
+- bw
+- cap
+- cel
+- cin
+- cine
+- cr2
+- cr3
+- crw
+- cs1
+- data
+- dc2
+- dcm
+- dcr
+- dds
+- desktop
+- dicom
+- dng
+- eps
+- erf
+- exr
+- fff
+- fit
+- fits
+- flc
+- fli
+- g3
+- gbp
+- gbr
+- gif
+- gih
+- hdr
+- heic
+- heif
+- hgt
+- ia
+- icb
+- ico
+- icon
+- iiq
+- im1
+- im24
+- im32
+- im8
+- j2c
+- j2k
+- jp2
+- jpc
+- jpe
+- jpeg
+- jpg
+- k25
+- kc2
+- kdc
+- mask
+- matte
+- mdc
+- mef
+- mos
+- mrw
+- nef
+- nrw
+- ora
+- orf
+- pat
+- pbm
+- pcc
+- pcx
+- pdf
+- pef
+- pfm
+- pgm
+- pix
+- png
+- pnm
+- ppm
+- ps
+- psd
+- psp
+- pspimage
+- pxn
+- qtk
+- ra
+- raf
+- ras
+- raw
+- rdc
+- rgb
+- rgba
+- rw2
+- rwl
+- sgi
+- sr2
+- srf
+- srw
+- sti
+- sun
+- svg
+- tga
+- tif
+- tiff
+- tub
+- vda
+- vst
+- webp
+- wmf
+- x3f
+- xbm
+- xcf
+- xcf.bz2
+- xcf.gz
+- xcfbz2
+- xcfgz
+- xcfxz
+- xpm
+- xwd
+ProductCode: GIMP-2_is1
+ReleaseDate: 2024-10-03
+AppsAndFeaturesEntries:
+- Publisher: The GIMP Team
+  DisplayName: GIMP 2.10.38-1
+  DisplayVersion: 2.10.38
```

Comment:
Don't like the idea of introducing `DisplayVersion` for the package. Because once DisplayVersion is added in one version manifest, it needs to be added for all others too to avoid version mapping errors. And in adding it in all others, we'd be adding unnecessary mapping code since PackageVersion would be equal to DisplayVersion in majority of manifests, and the field would serve no purpose.

Looking at the original issue, I don't think we can do much from WinGet side unless the publisher uses different DisplayVersions for each revision release. We retain the latest manifest out of all that share the same DisplayVersion